README.md

```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib).
```

Comment:
```suggestion
The OSIsoft connector uses the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) for application logging to the Ewon Flexy's realtime logs.
```

Reply:
I think this might clarify what this is doing a little more to the average user.